routes/Users.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const passport = require('passport');
+const bcrypt = require('bcrypt');
+const User = require("../models/user");
+require('../password-config');
+const app = express();
+const router = express.Router();
+
+app.use("/api", router);
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+router.post("/create", (req, res) => {
+
+    let { username, password } = req.body,
```

Comment:
Keep each declaration as separate statement. When you declare variables using one `let` and separating them by comma - you have to make extra actions to add one more variable.
```suggestion
    const { username, password } = req.body;
    const user = new User();
```